src/curl.c

```diff
@@ -480,6 +480,8 @@ CURLcode swupd_curl_set_basic_options(CURL *curl, const char *url)
 		goto exit;
 	}
 
+	curl_easy_setopt(curl, CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_2_0);
+
```

Comment:
That does not work; the NONE means HTTP/1.1
We need to explicitly give HTTP/2 permission.

2_PRIOR_KNOWLEDGE is also not a good option since that removed the potential for a fallback to HTTP/1.1
for when infrastructure (like proxies) are not HTTP/2 ready